### DISCUSSION TITLE
How to Check If array Is Empty in Controller

### DISCUSSION THREAD

**Discussion by yogigurjar** [livewire/livewire#8272] [1. Help]
I have a following code and want to check array is empty or not:
 
```
$bookingLists = Booking::where(['bookings.user_id' => $user_id,'bookings.status' => $bStatus ])->get();
            
   if(empty($bookingLists)){
            
        }else{
}
```

**Comment by bedo2991** [comment]
This looks like a Laravel question.
The `get() `method returns a [Collection](https://laravel.com/api/11.x/Illuminate/Database/Eloquent/Collection.html), therefore you can use `$bookingLists->isEmpty()`
